examples/AllTests/FEDemoTest.cpp

```diff
@@ -28,9 +28,9 @@
 #include "CppUTest/CommandLineTestRunner.h"
 #include "CppUTest/TestHarness.h"
 #include "CppUTest/TestRegistry.h"
-#include "CppUTestExt/IEEE754ExceptionsPlugin.h"
 
 #ifdef CPPUTEST_HAVE_FENV
+#include "CppUTestExt/IEEE754ExceptionsPlugin.h"
```

Comment:
This change shouldn't be needed, right? We should be able to safely include the header file. It is ok to keep it, but it shouldn't be because of a compiler error....